daemon/src/main/java/org/apache/maven/cli/DaemonMavenOptions.java

```diff
@@ -24,7 +24,6 @@
 import org.apache.maven.api.cli.mvn.MavenOptions;
 
 public interface DaemonMavenOptions extends MavenOptions {
-
```

Comment:
Am unsure about the existence of this class. DaemonMavenOptions was introduced to add new CLI options like "raw streams", but today, no new option in mvnd exists...

Reply:
That's kind expected, as mvnd specific options (such as `--stop`) are handled differently at the very beginning, or via system properties.